### PR TITLE
Made relation between the navBar and the accordion thank to the presentation's button

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -97,6 +108,7 @@ export default function NavBar() {
           </a>
         </div>
       </nav>
+      {accordion === true ? <Accordeon /> : null}
```
Comment (GeorgesFORTRY) [inline src/components/NavBar/NavBar.jsx]: no need to put "=== true", just "accordion ?" it's enough

**Review by GeorgesFORTRY — CHANGES_REQUESTED** [review]
a little change is needed

**Review by GeorgesFORTRY — DISMISSED** [review]
ok

**Review by GeorgesFORTRY — DISMISSED** [review]
ok

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by GeorgesFORTRY — DISMISSED** [review]
ok

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**